docs/concepts.rst

```diff
@@ -541,10 +541,12 @@ object types it's more efficient or more idiomatic to use a different format.
 There are also some types of objects that can't be pickled at all.  Bionic uses
 ``pickle`` by default, but handles some types specially: `Pandas
 <https://pandas.pydata.org/>`_ DataFrames are serialized in the `Parquet
-<https://parquet.apache.org/>`_ format, while `Pillow
-<https://pillow.readthedocs.io/en/stable/>`_ Images are serialized as `PNG
-<https://en.wikipedia.org/wiki/Portable_Network_Graphics>`_\ s.  You can
-explictly specify a serialization strategy for an entity by attaching a
+<https://parquet.apache.org/>`_ format, `Geopandas <https://geopandas.org/>`_
```

Comment:
I guess we should add the other formats we automatically handle as well: NumPy and Dask.

Also, it looks like it's "GeoPandas", not "Geopandas".